### PR TITLE
Add foc/search

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,6 +231,7 @@ Focus is on the specific task.
 
 - [ElasticSearch plugin](https://github.com/cakephp/elastic-search) - Alternative ORM using Elastic Search as its backend.
 - [Search plugin](https://github.com/CakeDC/search) - De-facto search form standard built on PRG pattern.
+- [Search pluggin](https://github.com/FriendsOfCake/search) - Search provides a search module for CakePHP applications.
```
Comment (jadb) [inline README.md]: Single 'g' to plugin ;)

Comment (ADmad) [inline README.md]: It helped differentiate with the one above :stuck_out_tongue:

Comment (dereuromark) [inline README.md]: Sure does :P

**Comment by dereuromark** [comment]
If you had changed the namespace to FOC/Search it would have also been a different plugin name now ;)

**Comment by ADmad** [comment]
Who knows, over time FOC's search might become de-facto :)

**Comment by dereuromark** [comment]
From the looks it sure sounds like it could happen. More objects, less arrays :)

**Pull request merged**